libp2p/muxers/mplex/lpchannel.nim

```diff
@@ -93,12 +93,45 @@ proc resetMessage(s: LPChannel) {.async.} =
   await s.conn.writeMsg(s.id, s.resetCode)
 
 proc resetByRemote*(s: LPChannel) {.async.} =
-  await allFutures(s.close(), s.closedByRemote())
+  let
+    f1 = awaitne s.close()
+    f2 = awaitne s.closedByRemote()
+  # NOTICE WE DO NOT RETHROW
+  # Those failures are not critical but still wait and warn about them!
+  if f1.failed:
+    let exc = f1.readError()
+    # EOF are likely to happen here
+    if exc.name != "LPStreamEOFError":
+      warn "Something went wrong during resetByRemote -> close",
+       failure = exc.name, msg = exc.msg
+  if f2.failed:
+    warn "Something went wrong during resetByRemote -> closedByRemote",
+     failure = f2.readError.name, msg = f2.readError.msg
+
   s.isReset = true
-  await s.cleanUp()
+
+  let
+    f3 = awaitne s.cleanUp()
+  # NOTICE WE DO NOT RETHROW
```

Comment:
this might seem very dramatic right now because we've had long discussions about it, but at the end of the day, it's not _that_ unusual to consciously decide not to do anything with an error ;)